nullary_ho_func_error.m:022: In clause for predicate `nullary_ho_func_error.main/2':
nullary_ho_func_error.m:022:   in argument 1 of call to predicate `print/3':
nullary_ho_func_error.m:022:   in argument 1 of functor `apply_nullary_func/1':
nullary_ho_func_error.m:022:   type error in unification of argument
nullary_ho_func_error.m:022:   and constant `pi'.
nullary_ho_func_error.m:022:   argument has type `((func) = T)',
nullary_ho_func_error.m:022:   constant `pi' has type `float'.
For more information, try recompiling with `-E'.
